src/Language/PureScript/Ide/Rebuild.hs

```diff
@@ -154,11 +156,25 @@ shushProgress :: P.MakeActions P.Make -> MakeActionsEnv -> P.MakeActions P.Make
 shushProgress ma _ =
   ma { P.progress = \_ -> pure () }
 
--- | Stops any kind of codegen (also silences errors about missing or unused FFI
--- files though)
```

Comment:
@kRITZCREEK I was led by this comment. Anyway, the dictionary lookup should not be very expensive (which is the only overhead added to modules without ffi), `checkForeignDecls` is though, but it's also useful.  As I understand, now ffi errors will magically ;) disappear when a module is rebuild.

Reply:
The comment is misleading, yeah. It's technically not wrong, but we don't care about any errors that might be generated here anyway because we've already gotten all the errors and sent them off to the editor at this point. I'd be totally fine if you just ripped the parenthesized part out.

Reply:
The first build: https://github.com/purescript/purescript/blob/fb8daff44cce61fb404d30695065769fef582401/src/Language/PureScript/Ide/Rebuild.hs#L73-L74

doesn't `shushCodegen`.